packages/app/src/runs/RunCard.vue

```diff
@@ -11,42 +11,49 @@
       :data-cy="`run-card-icon-${run.status}`"
     >
       <template #header>
-        {{ run.commitInfo?.summary }}
+        <span class="font-semibold mr-8px whitespace-pre-wrap">{{ run.commitInfo?.summary }}</span>
+        <span
+          v-for="tag in tags"
+          :key="tag"
+          class="rounded-md font-semibold border-gray-200 border-1px text-xs mr-8px px-4px text-gray-700"
+          data-cy="run-tag"
+        >{{ tag }}</span>
       </template>
       <template #description>
-        <div class="flex">
-          <span
+        <ul class="flex flex-wrap text-sm text-gray-700 gap-8px items-center whitespace-nowrap children:flex children:items-center">
+          <li
             v-if="run.commitInfo?.authorName"
-            class="flex mr-3 items-center"
             data-cy="run-card-author"
           >
             <i-cy-general-user_x16
-              class="mr-1 icon-dark-gray-500 icon-light-gray-200 icon-secondary-light-gray-200"
+              class="mr-1 icon-dark-gray-500 icon-light-gray-100 icon-secondary-light-gray-200"
               data-cy="run-card-avatar"
             />
-            <span class="font-light text-sm text-gray-500">
-              {{ run.commitInfo.authorName }}
-            </span>
-          </span>
-          <span
+            {{ run.commitInfo.authorName }}
```

Comment:
```suggestion
            {{ run.commitInfo.authorName }}
            <span class="sr-only">Commit Author:</span>{{ run.commitInfo.authorName }}
```

The icons and visual context help explain what each fragment of UI represents, it would be worthwhile to add an "sr-only" label like this, to each of these fields for screen reader access.

Reply:
EVERYBODY GETS AN SR-ONLY TAG: [4af1371](https://github.com/cypress-io/cypress/pull/21410/commits/4af137176e6bc1e15c9ef8bb55b553d2a01647d1)